### PR TITLE
Adding Test function and log

### PR DESCRIPTION
**Pull request by Hariprasadm1408**

### CODE REVIEW

File: test/test_create_floor_array.c [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+struct passenger{
+bool	in_elevator; //!< Stores whether passegner is in elevator. 
+unsigned int	arrival_time; //!< stores arrival time of the passenger
+unsigned int	arrival_floor; //!< stores the arrival_floor of the passenger
+unsigned int	dest_floor; //!< stores the destination floor of the passenger
+unsigned int 	id; //!< stores the passenger_id of the passenger
+struct passenger*next; //! stores address of the next node in case of creation of queues
+};
+
+struct passenger ** create_floor_array(int no_of_floors)
```
Comment (MaazJamal) [inline test/test_create_floor_array.c]: Make these functions static. so they wont interfere with the other functions we have included

File: test/test_input_after_stop.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	{
+		char temp[100];
+        sprintf(temp, "PASS: Valid Input and returned successfully, Expected Output: 1 or 0, Output:1 or 0 \n");
+        printf(temp);
+        fprintf(test_results, temp);
+	}
+	return 0;
+}
+
+int input_after_stop()
```
Comment (MaazJamal) [inline test/test_input_after_stop.c]: This function should be made static aswell.

**Review by MaazJamal — APPROVED** [review]
Looks fine. now

**Review by divyalakshmi-padmanaban — APPROVED** [review]
With static functions,looks fine to me.

**Review by ritika2 — APPROVED** [review]
Looks good

**Pull request merged**